Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.Properties.cs

```diff
@@ -67,7 +68,7 @@ public static void SetMask(DependencyObject obj, string value)
         /// </summary>
         /// <param name="obj">TextBox control</param>
         /// <returns>placeholder value</returns>
-        public static string GetPlaceHolder(DependencyObject obj)
+        public static string GetPlaceHolder(TextBox obj)
```

Comment:
Can you change the sample to ensure that we have use both x:Bind and Binding.. I have a feeling TextBox is going to bust x:Bind